src/components/Input/index.js

```diff
@@ -0,0 +1,40 @@
+import React, { useState } from "react"
+import PropTypes from "prop-types"
+import { InputCls } from "./styles"
+import { cx } from "emotion"
+
+Input.propTypes = {
+  className: PropTypes.string,
+  onChange: PropTypes.func,
+  label: PropTypes.string,
+  value: PropTypes.string,
+  disabled: PropTypes.bool,
+  errorMessage: PropTypes.string
+}
+
+function Input({ className, onChange, label, value, disabled, errorMessage }) {
+  const [inputValue, setInputValue] = useState("")
+  const [isDisabled, setIsDisabled] = useState(false)
+  const [error, setError] = useState(false)
```

Comment:
If there is an error to the field, the errorMessage prop has a value. If not then the errorMessage prop is empty. No need to keep an internal error state